buildSrc/src/main/kotlin/themes/GroupStyling.kt

```diff
@@ -1,4 +1,4 @@
-package com.markskelton.settings
+package themes
```

Comment:
Oh the sadness 😆

Reply:
![I'll be back](https://media.giphy.com/media/JDKxRN0Bvmm2c/giphy.gif)